### PR TITLE
Load script/style asynchronously.

### LINKED ISSUE #10306: Load script/style asynchronously.
https://support.abp.io/QA/Questions/1954/MVC-Bundling--Minification--Abount-Google-Page-Speed

 **<link rel="preload" href=**
**<script defer src="**

https://github.com/abpframework/abp/blob/dev/framework/src/Volo.Abp.AspNetCore.Mvc.UI.Bundling/Volo/Abp/AspNetCore/Mvc/UI/Bundling/TagHelpers/AbpTagHelperStyleService.cs#L46

https://github.com/abpframework/abp/blob/dev/framework/src/Volo.Abp.AspNetCore.Mvc.UI.Bundling/Volo/Abp/AspNetCore/Mvc/UI/Bundling/TagHelpers/AbpTagHelperScriptService.cs#L46

### PR DESCRIPTION
**Pull request by maliming**
Resolve #10306

```cs
<abp-script defer="true" src="/Pages/Index.js" />

<abp-script-bundle defer="true">
    <abp-script src="/Pages/Index.js" />
</abp-script-bundle>

Configure<AbpTagHelperScriptStyleLoadingOptions>(options =>
{
    options.PreloadStyles.Add("/__bundles/Basic.Global");
});
```

### CODE REVIEW

File: framework/src/Volo.Abp.AspNetCore.Mvc.UI.Bundling/Volo/Abp/AspNetCore/Mvc/UI/Bundling/TagHelpers/AbpTagHelperScriptStyleLoadingOptions.cs [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+namespace Volo.Abp.AspNetCore.Mvc.UI.Bundling.TagHelpers
+{
+    public class AbpTagHelperScriptStyleLoadingOptions
+    {
+        public bool GlobalDeferScript { get; set; }
+
+        public List<string> DeferScripts { get; }
+
+        public bool GlobalPreloadStyle { get; set; }
```
Comment (hikalkan) [inline framework/src/Volo.Abp.AspNetCore.Mvc.UI.Bundling/Volo/Abp/AspNetCore/Mvc/UI/Bundling/TagHelpers/AbpTagHelperScriptStyleLoadingOptions.cs]: `GlobalPreloadStyle` is not a good name for a boolean. We can rename it to `PreloadStylesByDefault`.

File: framework/src/Volo.Abp.AspNetCore.Mvc.UI.Bundling/Volo/Abp/AspNetCore/Mvc/UI/Bundling/TagHelpers/AbpTagHelperScriptStyleLoadingOptions.cs [hunk 2]
```diff
@@ -0,0 +1,24 @@
+﻿using System.Collections.Generic;
+
+namespace Volo.Abp.AspNetCore.Mvc.UI.Bundling.TagHelpers
+{
+    public class AbpTagHelperScriptStyleLoadingOptions
```
Comment (hikalkan) [inline framework/src/Volo.Abp.AspNetCore.Mvc.UI.Bundling/Volo/Abp/AspNetCore/Mvc/UI/Bundling/TagHelpers/AbpTagHelperScriptStyleLoadingOptions.cs]: I think we can remove this class (I didn't like the name much), and move options to `AbpBundlingOptions.cs`.

File: framework/src/Volo.Abp.AspNetCore.Mvc.UI.Bundling/Volo/Abp/AspNetCore/Mvc/UI/Bundling/TagHelpers/AbpTagHelperScriptStyleLoadingOptions.cs [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    {
+        public bool GlobalDeferScript { get; set; }
```
Comment (hikalkan) [inline framework/src/Volo.Abp.AspNetCore.Mvc.UI.Bundling/Volo/Abp/AspNetCore/Mvc/UI/Bundling/TagHelpers/AbpTagHelperScriptStyleLoadingOptions.cs]: We can rename this to `DeferScriptsByDefault`.

**Review by hikalkan — CHANGES_REQUESTED** [review]

**Comment by hikalkan** [comment]
Please create an issue about the documentation update and send a PR for it separately.

**Review by realLiangshiwei — APPROVED** [review]

**Review by hikalkan — APPROVED** [review]

**Pull request merged**